content/news/2023-10-01-r-ladies-global-and-r-ladies-chicago-reception-at-posit-conf/index.en.md

```diff
@@ -0,0 +1,221 @@
+slido. The panel included Athanasia Mowinckel, Yanina Bellini Saibene,
+and Riva Quiroga from the leadership team as well as R-Ladies chapter
+organizers María Nanton (R-Ladies Buenos Aires), Stephanie Orellana
+(R-Ladies Santiago) and Angela Li (R-Ladies Chicago). Thank you to all
+of our panelists for sharing their insights and experiences.
+
+The question and answers below are not presented as they were discussed
+chronologically, but more thematically in terms of discussion versus
+operational/procedural questions.
+
+# Discussion questions
+
+## What are the main benefits of R-Ladies?
+
+*Angela:* Improving my data science skills in a community that was
+really supportive, which then quickly cascaded into talking to people in
+different sectors doing different things that you didn't yet know.
+Angela and Ola are stepping down as R-Ladies Chicago organizers, and
+would be amazing to give those benefits to someone else to make
+connections and build their careers.
+
+*Maria:* A space where you can grow in a supportive environment.
+Professionally, I wouldn't have the job that I have right now without
+R-Ladies. It has been a hugely positive experience, and it has been
+amazing to give back as an organizer.
+
+*Stephanie:* R-Ladies changed my life, I am a super introvert and felt
+comfortable here; I have learned a lot and I made really good friends. I
+also like that it is an international community. I didn't have
+international friends before this and now I have friends in other
+R-Ladies in Spanish and they ask me to share my knowledge with them.
+
+*Riva:* It is a great community to be an introvert in; R-Ladies also
+changed my life. You have fun, you learn, and it helps with imposter
+syndrome. I don't have words to express how it changed my life.
+
+*Mo:* It is changing individual lives, but it is also changing an entire
+community. We are at tech conference and there are so many women here -
+this of course has to do with the R community that we are in and this R
+community is influenced that R-Ladies is breaking stereotypes and making
+spaces for all of us. That has been a huge success for us,
+as interest for R-Ladies chapters quickly developed  and
+became popular very fast. The organization has made space for all of us
+in data science fields. It benefits us and the entire community. There
+are amazing initiatives going on as well. By establishing the idea of creating more diversity
+in gender representation in our community (here: the larger R-community), other aspects of
+diversity have become more highlighted also. The efforts on translation of content and even R
+it self, R-Ladies, is one of these.
+R-Ladies in Latin American is making a huge
+impact on a larger scale and all the personal stories are fantastic to
+listen to.
+
+## Will R-Ladies ever re-brand to a name that is more inclusive to R users of marginalized genders?
+
+*Yani:* Yes. As of now our plan is to open a discussion within the
```

Comment:
```suggestion
*Yani:* Yes, it is under consideration. As of now our plan is to open a discussion within the
```

Reply:
I agree, I like the change. When I answered, it was to the consideration of changing the name.  The decision to change or not will come from the discussion.